image_width: 1280
image_height: 800
camera_name: narrow_stereo/left
camera_matrix:
  rows: 3
  cols: 3
  data: [967.5367369912963, 0, 616.4853232338544, 0, 964.186989950682, 339.4439554192681, 0, 0, 1]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [-0.3437310668560763, 0.1276143644624977, 0.001372635071413418, -0.001696329329827227, 0]
rectification_matrix:
  rows: 3
  cols: 3
  data: [0.99997110271294, -0.003269844721013331, 0.006863079087935261, 0.00328053035115901, 0.9999934235266223, -0.001546293672153857, -0.006857977812877322, 0.001568763527712109, 0.9999752532544554]
distortion_coefficients:
  rows: 3
  cols: 4
  data: [921.45692061762, 0, 646.8515548706055, 0, 0, 921.45692061762, 331.6418113708496, 0, 0, 0, 1, 0]
